docs/excel/excel-add-ins-ranges.md

```diff
@@ -585,9 +585,13 @@ Excel.run(function (context) {
 ```
 
 *Before the preceeding function has been run.*
+
 ![Data in Excel before range’s copy method has been run.](../images/excel-range-copyfrom-skipblanks-before.png)
+
 *After the preceeding function has been run.*
-![Data in Excel after range’s copy method has been run.](../images/ excel-range-copyfrom-skipblanks-after.png)
+
+![Data in Excel after range’s copy method has been run.](../images/excel-range-copyfrom-skipblanks-after.png)
+
 `transpose` determines whether the data is algebraically transposed into the source location. A transposed range has its rows and columns switched; flipping the cells along the main diagonal. 
```

Comment:
Not sure you should use transpose in a definition of transpose?

Reply:
I would argue I'm defining a parameter called "transpose" here and specifying what the specific usage of that word is.  There's [non-algebra definitions](https://www.merriam-webster.com/dictionary/transpose) too :-)